notebooks/postgresql2bq/PostgreSqlToBigQuery_parameterize_script.py

```diff
@@ -0,0 +1,169 @@
+            dest=constants.POSTGRESQL_PASSWORD,
+            required=True,
+            help='POSTGRESQL password'
+        )
+
+        parser.add_argument(
+            f'--{constants.POSTGRESQL_DATABASE_ARG}',
+            dest=constants.POSTGRESQL_DATABASE,
+            required=True,
+            help='POSTGRESQL database name'
+        )
+
+        parser.add_argument(
+            f'--{constants.POSTGRESQL_TABLE_LIST_ARG}',
+            dest=constants.POSTGRESQL_TABLE_LIST,
+            required=False,
+            default='',
+            help='POSTGRESQL table list to migrate. '
+            'Leave empty for migrating complete database else provide tables as \"table1,table2\"'
+        )
+
+        parser.add_argument(
+            f'--{constants.POSTGRESQL_SCHEMA_LIST_ARG}',
+            dest=constants.POSTGRESQL_SCHEMA_LIST,
+            required=False,
+            default='',
+            help='POSTGRESQL schema list to migrate. '
+            'Only Migrate tables associated with the provided schema list'
+        )
+
+       
+        parser.add_argument(
+            f'--{constants. BIGQUERY_DATASET_ARG}',
+            dest=constants. BIGQUERY_DATASET,
+            required=True,
+            help=' BIGQUERY dataset name'
+        )
+
+        parser.add_argument(
+            f'--{constants. BIGQUERY_MODE_ARG}',
+            dest=constants. BIGQUERY_MODE,
+            default='overwrite',
+            required=True,
+            help='output write mode (Default: overwrite)'
+        )
+
+        parser.add_argument(
+            f'--{constants.MAX_PARALLELISM_ARG}',
+            dest=constants.MAX_PARALLELISM,
+            type=int,
+            default=5,
+            required=False,
+            help='Maximum number of tables that will migrated parallelly (Default: 5)'
+        )
+
+        known_args: argparse.Namespace
+        known_args, _ = parser.parse_known_args()
+
+        return vars(known_args)
+
+
+    def get_env_var(self, parameters) ->  Dict[str, Any]:
+        """
+        Get the environment variables.
+        """
+      
+        parameters[constants.PROJECT] = environ[constants.GCP_PROJECT]
+        parameters[constants.REGION] = environ[constants.REGION]
+        parameters[constants.GCS_STAGING_LOCATION] = environ[constants.GCS_STAGING_LOCATION]
+        parameters[constants.SUBNET] = environ[constants.SUBNET] if constants.SUBNET in environ else ""
+        parameters[constants.SERVICE_ACCOUNT] = environ[constants.SERVICE_ACCOUNT] if constants.SERVICE_ACCOUNT in environ else ""
+        parameters[constants.IS_PARAMETERIZED] = True
+        
+
+        return parameters
+
+
+    def run(self, args: Dict[str, Any]) -> None:
+        """
+        Run the notebook.
+        """
+
+        # Convert comma separated string to list
+        args[constants.POSTGRESQL_TABLE_LIST] = list(
+            map(str.strip, args[constants.POSTGRESQL_TABLE_LIST].split(","))
+        )
```

Comment:
Looks like `args[constants.POSTGRESQL_SCHEMA_LIST]` must also be converted from a string to list, similar to what has been done for `POSTGRESQL_TABLE_LIST`